src/locales/en/main.json

```diff
@@ -17,6 +17,7 @@
   "replay": "Replay",
   "settings": "Settings",
   "skipmessage": "Skip ad in xx",
+  "sharing": "Share post",
```

Comment:
we are not sharing a post, right?

Reply:
We are not, but that's what it says in the designs :)